networkx/readwrite/json_graph/cytoscape.py

```diff
@@ -11,16 +11,42 @@ def cytoscape_data(G, attrs=None):
     Parameters
     ----------
     G : NetworkX Graph
-
+        The graph to convert to cytoscape format
+    attrs : dict or None (default=None)
+        A dictionary containing the keys 'name' and 'ident' which are mapped to
+        the 'name' and 'id' node elements in cyjs format. All other keys are
+        ignored. Default is `None` which results in the default mapping
+        ``{name="name", ident="id"}``.
 
     Returns
     -------
     data: dict
         A dictionary with cyjs formatted data.
+
     Raises
     ------
     NetworkXError
-        If values in attrs are not unique.
+        If values in `attrs` are not unique.
+
+    See Also
+    --------
+    cytoscape_graph - convert a dictionary in cyjs format to a graph
+
+    References
+    ----------
+    .. [1] Cytoscape user's manual:
+       http://manual.cytoscape.org/en/stable/index.html
+
+    Examples
+    --------
+    >>> G = nx.path_graph(2)
+    >>> nx.cytoscape_data(G)  # doctest: +SKIP
```

Comment:
This comes down to REPL formatting; in IPython you get the nice dict formatting, but with the standard REPL you don't get the automatic newlines for every key-value pair. I decided that readability was more important than the doctest so I went with the IPython-formatted output. It could be modified to e.g. use prettyprint or `print(f"{k}: {v}") for k, v in data`, but then the example becomes a little less clear